### PR TITLE
Add test for dryRun checking saleMessageType argument

### PR DESCRIPTION
**Pull request by vlaskhilkevich**

### CODE REVIEW

**Review by singh-sukhvir — COMMENTED** [review]
looks gd, just a minor comment

File: src/test/java/org/killbill/billing/plugin/vertex/VertexTaxCalculatorTest.java [hunk 1]
```diff
@@ -175,40 +161,12 @@ public void testCompute() throws Exception {
...
-
-        List<OwnerResponseLineItemType> responseLineItemTypeList = Arrays.asList(responseLineItem);
-        given(apiResponseData.getLineItems()).willReturn(responseLineItemTypeList);
-        given(taxResponse.getData()).willReturn(apiResponseData);
-        given(vertexApiClient.calculateTaxes(any(SaleRequestType.class))).willReturn(taxResponse);
-
         //when
-        vertexTaxCalculator.compute(
-                account, invoice, true, pluginProperties, tenantContext);
+        vertexTaxCalculator.compute(account, invoice, true, Collections.emptyList(), tenantContext);
```
[the remark this answers is not in the document]
Comment (vlaskhilkevich) [reply src/test/java/org/killbill/billing/plugin/vertex/VertexTaxCalculatorTest.java]: FIxed

File: src/test/java/org/killbill/billing/plugin/vertex/VertexTaxCalculatorTest.java [hunk 2]
```diff
@@ -143,24 +108,45 @@ public void testCompute() throws Exception {
...
+
+        //then
+        verify(vertexDao).addResponse(any(UUID.class), any(UUID.class), anyMap(), any(ApiSuccessResponseTransactionResponseType.class), any(DateTime.class), any(UUID.class));
+        assertEquals(0, result.size());
+    }
+
+    @Test(groups = "fast")
+    public void testCompute() throws Exception {
+        //when
+        List<InvoiceItem> result = vertexTaxCalculator.compute(account, invoice, false, Collections.emptyList(), tenantContext);
```
[the remark this answers is not in the document]
Comment (vlaskhilkevich) [reply src/test/java/org/killbill/billing/plugin/vertex/VertexTaxCalculatorTest.java]: Fixed

**Review by singh-sukhvir — APPROVED** [review]
looks gd

**Pull request merged**